index.html

```diff
@@ -17,6 +17,9 @@
   <title>SHOP</title>
 
   <link rel="shortcut icon" sizes="32x32" href="/images/shop-icon-32.png">
+  <meta name="google-signin-client_id" content="569649340112-da0ku71mend03mdsu9h7skmv33r2qkg2.apps.googleusercontent.com">
+  <script src="https://apis.google.com/js/platform.js"></script>
+  <script src="/bower_components/fetch/fetch.js"></script>
```

Comment:
@blasten, [fetch](https://github.com/github/fetch) is a polyfill, and since all modern browsers implement the `window.fetch` function natively, no code from this project actually takes any effect in these browsers. Please note that [older browsers](http://caniuse.com/#feat=promises) requires `Promise` polyfill for `fetch` work.